files/default/sql/ddl/updates/undo/2.6.0__undo.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `hopsworks`.`feature_group` DROP COLUMN `stream_feature_group_id`;
+ALTER TABLE `hopsworks`.`feature_group` DROP KEY `stream_feature_group_fk`;
+ALTER TABLE `hopsworks`.`feature_group` DROP CONSTRAINT `stream_feature_group_fk`;
```

Comment:
Did you test this? I believe you have to drop the Key and Constraint first, and drop the column last.